watertap/examples/flowsheets/full_treatment_train/flowsheet_components/chemistry/PreTreatment_Simple_Softening.py

```diff
@@ -46,7 +46,7 @@
 # Import pyomo methods to check the system units
 
 
```

Comment:
This file (PreTreatment_Simple_Softening.py) is removed by PR #284 (redundant and unused in full flowsheet)